lambdas/call-ticf-cri/src/test/java/uk/gov/di/ipv/core/callticfcri/pact/ticfCri/ContractTest.java

```diff
@@ -0,0 +1,636 @@
+@ExtendWith(PactConsumerTestExt.class)
+@ExtendWith(MockitoExtension.class)
+@PactTestFor(providerName = "TicfCriProvider")
+@MockServerConfig(hostInterface = "localhost", port = "1234")
+class ContractTest {
+    private static final String API_PATH = "/risk-assessment";
+    private static final String PRIVATE_API_KEY = "dummyApiKey";
+    private static final String PRIVATE_TICF_SIGNING_KEY =
+            """
+                    {"kty":"EC","d":"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU","crv":"P-256","x":"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM","y":"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04"}
+            """;
+    private static final String APPLICATION_JSON = "application/json";
+    private static final Clock CURRENT_TIME =
+            Clock.fixed(Instant.parse("2099-01-01T00:00:00.00Z"), ZoneOffset.UTC);
+    private static PactJwtBuilder dvlaVcJwtHelper;
+    private static PactJwtBuilder passportVcJwtHelper;
+    private static PactJwtBuilder dvlaWithCiVcJwtHelper;
+    private static PactJwtBuilder emptyTicfVcJwtHelper;
+    private static PactJwtBuilder interventionTicfVcJwtHelper;
+    private static PactJwtBuilder noInterventionTicfVcJwtHelper;
+    private static PactJwtBuilder noInterventionWithWarningsTicfVcJwtHelper;
+
+    @Mock ConfigService mockConfigService;
+
+    @BeforeAll
+    public static void setup() throws IOException {
+        dvlaVcJwtHelper = PactJwtBuilder.fromPath("/dvlaVc");
+        passportVcJwtHelper = PactJwtBuilder.fromPath("/passportVc");
+        dvlaWithCiVcJwtHelper = PactJwtBuilder.fromPath("/dvlaWithCiVc");
+        emptyTicfVcJwtHelper = PactJwtBuilder.fromPath("/ticfVc/empty");
+        interventionTicfVcJwtHelper = PactJwtBuilder.fromPath("/ticfVc/intervention");
+        noInterventionTicfVcJwtHelper = PactJwtBuilder.fromPath("/ticfVc/noIntervention");
+        noInterventionWithWarningsTicfVcJwtHelper =
+                PactJwtBuilder.fromPath("/ticfVc/noInterventionWithWarnings");
+    }
+
+    @Pact(provider = "TicfCriProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact validPassportVcReturnsVcWithNoInterventions(
+            PactDslWithProvider builder) {
+        return builder.given("dummyApiKey is a valid api key")
+                .given(
+                        "Provided VC can be validated with {\"kty\":\"EC\",\"crv\":\"P-256\",\"x\":\"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM\",\"y\":\"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04\"}")
+                .given("TICF VC has no interventions or warnings")
+                .given("TICF VC risk assessment has id riskAssessmentId")
+                .given("TICF VC issuer is https://ticf.account.gov.uk")
+                .given(
+                        "TICF VC is signed with {\"kty\":\"EC\",\"d\":\"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU\",\"crv\":\"P-256\",\"x\":\"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM\",\"y\":\"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04\"}")
```

Comment:
Not sure we need this - we do need to know that our example VC is signed with this key, but the TICF signature will be ignored in the tests.